src/serialization/writer.py

```diff
@@ -81,11 +103,18 @@ def main(argv):
 
     #find all the pickles
     pickles = glob.glob('{}*.p'.format(pickleDir))
+    with open("/zooper1/fontbakers/font_caps_protos",'r') as fo: 
```

Comment:
This needs to be yapfed. Suspicious that #57 doesn't fix this. cc @ArianaFreitag 

```suggestion
    with open("/zooper1/fontbakers/font_caps_protos", 'r') as fo: 
```

Reply:
Followed up with Ariana separately.